src/app/components/publiclists/teamlist/teamlist.component.scss

```diff
@@ -161,3 +161,7 @@ span.link-team-user {
     margin-top: 30px;
   }
 }
+
+#teamdetailslayout {
+  margin-left: 40px;
```

Comment:
Why this change?

Reply:
Just a part of old experiment . i removed it